pkg/cli/debug/debug.go

```diff
@@ -81,13 +82,17 @@ var (
 
 		You may invoke other types of objects besides pods - any controller resource that creates
 		a pod (like a deployment, build, or job), objects that can host pods (like nodes), or
-		resources that can be used to create pods (such as image stream tags).
+		resources that can be used to create pods (such as image stream tags), or simply pass
+		'--image=IMAGE' to start a simple shell session in an image with a shell program
 
 		The debug pod is deleted when the remote command completes or the user interrupts
 		the shell.
 	`)
 
 	debugExample = templates.Examples(`
+		# Start a shell session into a pod using the OpenShift tools image
+		oc debug
```

Comment:
Why not just use `oc run --image=X`, there's nothing debug specific in doing this like so.

Reply:
Debug is better than run because debug is solving a different problem - debug is "how do i get a shell in an environment efficiently".  Run is about "how do i actually launch a pod".  The two are not equal.  And run frankly is a ux nightmare, while debug is about making opinionated choices to get that experience cleanly.

Reply:
Fair, I'm not a fan of run either :sweat: